packages/playground/src/components/pull-request/utils.tsx

```diff
@@ -201,3 +203,20 @@ export const getPrState = (is_draft?: boolean, merged?: number | null, state?: s
     return { icon: 'pr-open', text: 'Open', theme: 'success' }
   }
 }
+
+export const extractInfoFromRuleViolationArr = (ruleViolationArr: TypesRuleViolations[]) => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const tempArray: any[] = ruleViolationArr?.flatMap(
```

Comment:
Do we not know the type here? Better to use `unknown` instead.

Reply:
done